docs/concepts/projects/search.md

```diff
@@ -18,6 +18,27 @@ However, it may be preferable to have a stronger governance framework for ML ass
 
 ## Lineage
 
-Hopsworks tracks the lineage (or provenance) of ML assets automatically for you. You can see what features are used in which feature view or training dataset. You can see what training dataset was used to train a given model. For assets that are managed outside of Hopsworks, there is support for the explicit definition of lineage dependencies.
+Hopsworks tracks lineage in two separate ways:
+* implicit
+* explicit
+
+Implicit provenance means that the user doesn't have to change his code and the system will determine links between different artifacts based on their usage(read/write) within the context of the same application. 
+
+Explicit provenance has the user, explicitly marking the links between artifacts by performing additional method calls within their code.
```

Comment:
I think this sentence can be streamlined a bit - maybe "requires the user to explicitly mark"?